azurelinuxagent/common/cgroups.py

```diff
@@ -282,7 +288,7 @@ def collect_all_tracked():
         limits = {}
 
         for cgroup_name, collector in CGroupsTelemetry._tracked.copy().items():
-            cgroup_name = cgroup_name if cgroup_name else WRAPPER_CGROUP_NAME
+            cgroup_name = cgroup_name if cgroup_name else WRAPPER_CGROUP_TRACKED_NAME
```

Comment:
I thought we did not want to track the wrapper

Reply:
Since all the setup was already in place, I didn't want to introduce bigger changes by removing it because we can always choose to ignore it. But makes sense, it's really confusing having it. I'll remove it.